install.sh

```diff
@@ -371,9 +371,10 @@ get_version() {
         esac
     done
 
-    if [[ $_version == [0-9]* ]]; then
+    if echo "$_version" | grep -qE '^[0-9]+'; then
```

Comment:
If `if` doesn't work, pretty sure `case` will, but this is fine too.
```suggestion
    if echo "$_version" | grep -q '^[0-9]'; then
```